### ISSUE TITLE
Pagination and generate_sitemap.rb

### ISSUE THREAD

**Issue by bhardin** [recurser/jekyll-plugins#16]
When adding pagination to jekyll, generate_sitemap breaks. The following error is returned.

```
$ jekyll --server
/Users/bhardin/.rvm/rubies/ruby-1.9.3-p0/lib/ruby/site_ruby/1.9.1/rubygems/custom_require.rb:55:in `require': iconv will be deprecated in the future, use String#encode instead.
Configuration from /Users/bhardin/Documents/projects/blog/_config.yml
Building site: /Users/bhardin/Documents/projects/blog -> /Users/bhardin/Documents/projects/blog/_site
/Users/bhardin/Documents/projects/blog/_plugins/generate_sitemap.rb:85:in `mtime': No such file or directory - /Users/bhardin/Documents/projects/blog/page2/index.html (Errno::ENOENT)
        from /Users/bhardin/Documents/projects/blog/_plugins/generate_sitemap.rb:85:in `block in generate_content'
        from /Users/bhardin/Documents/projects/blog/_plugins/generate_sitemap.rb:83:in `each'
        from /Users/bhardin/Documents/projects/blog/_plugins/generate_sitemap.rb:83:in `generate_content'
        from /Users/bhardin/Documents/projects/blog/_plugins/generate_sitemap.rb:60:in `block in generate'
        from /Users/bhardin/Documents/projects/blog/_plugins/generate_sitemap.rb:58:in `open'
        from /Users/bhardin/Documents/projects/blog/_plugins/generate_sitemap.rb:58:in `generate'
        from /Users/bhardin/.rvm/gems/ruby-1.9.3-p0/gems/jekyll-0.11.2/lib/jekyll/site.rb:184:in `block in generate'
        from /Users/bhardin/.rvm/gems/ruby-1.9.3-p0/gems/jekyll-0.11.2/lib/jekyll/site.rb:183:in `each'
        from /Users/bhardin/.rvm/gems/ruby-1.9.3-p0/gems/jekyll-0.11.2/lib/jekyll/site.rb:183:in `generate'
        from /Users/bhardin/.rvm/gems/ruby-1.9.3-p0/gems/jekyll-0.11.2/lib/jekyll/site.rb:39:in `process'
        from /Users/bhardin/.rvm/gems/ruby-1.9.3-p0/gems/jekyll-0.11.2/bin/jekyll:250:in `<top (required)>'
        from /Users/bhardin/.rvm/gems/ruby-1.9.3-p0/bin/jekyll:19:in `load'
        from /Users/bhardin/.rvm/gems/ruby-1.9.3-p0/bin/jekyll:19:in `<main>'
```

Issue in Jekyll: https://github.com/mojombo/jekyll/issues/501

**Comment by ghost** [comment]
I think the issue is that the directory, `page2`, is found inside `_site` but `generate_sitemap.rb` is looking for it inside the project folder.

**Comment by recurser** [comment]
I _think_ this should be fixed in v0.2.0 - I've never actually managed to get pagination working so haven't seen this, but a couple of patches have been applied which I .think should fix it.
